mmdet3d/structures/det3d_data_sample.py

```diff
@@ -162,73 +164,88 @@ def gt_instances_3d(self) -> InstanceData:
         return self._gt_instances_3d
 
     @gt_instances_3d.setter
-    def gt_instances_3d(self, value: InstanceData):
+    def gt_instances_3d(self, value: InstanceData) -> None:
         self.set_field(value, '_gt_instances_3d', dtype=InstanceData)
 
     @gt_instances_3d.deleter
-    def gt_instances_3d(self):
+    def gt_instances_3d(self) -> None:
         del self._gt_instances_3d
 
     @property
     def pred_instances_3d(self) -> InstanceData:
         return self._pred_instances_3d
 
     @pred_instances_3d.setter
-    def pred_instances_3d(self, value: InstanceData):
+    def pred_instances_3d(self, value: InstanceData) -> None:
         self.set_field(value, '_pred_instances_3d', dtype=InstanceData)
 
     @pred_instances_3d.deleter
-    def pred_instances_3d(self):
+    def pred_instances_3d(self) -> None:
         del self._pred_instances_3d
 
     @property
     def pts_pred_instances_3d(self) -> InstanceData:
         return self._pts_pred_instances_3d
 
     @pts_pred_instances_3d.setter
-    def pts_pred_instances_3d(self, value: InstanceData):
+    def pts_pred_instances_3d(self, value: InstanceData) -> None:
         self.set_field(value, '_pts_pred_instances_3d', dtype=InstanceData)
 
     @pts_pred_instances_3d.deleter
-    def pts_pred_instances_3d(self):
+    def pts_pred_instances_3d(self) -> None:
         del self._pts_pred_instances_3d
 
     @property
     def img_pred_instances_3d(self) -> InstanceData:
         return self._img_pred_instances_3d
 
     @img_pred_instances_3d.setter
-    def img_pred_instances_3d(self, value: InstanceData):
+    def img_pred_instances_3d(self, value: InstanceData) -> None:
         self.set_field(value, '_img_pred_instances_3d', dtype=InstanceData)
 
     @img_pred_instances_3d.deleter
-    def img_pred_instances_3d(self):
+    def img_pred_instances_3d(self) -> None:
         del self._img_pred_instances_3d
 
     @property
     def gt_pts_seg(self) -> PointData:
         return self._gt_pts_seg
 
     @gt_pts_seg.setter
-    def gt_pts_seg(self, value: PointData):
+    def gt_pts_seg(self, value: PointData) -> None:
         self.set_field(value, '_gt_pts_seg', dtype=PointData)
 
     @gt_pts_seg.deleter
-    def gt_pts_seg(self):
+    def gt_pts_seg(self) -> None:
         del self._gt_pts_seg
 
     @property
     def pred_pts_seg(self) -> PointData:
         return self._pred_pts_seg
 
     @pred_pts_seg.setter
-    def pred_pts_seg(self, value: PointData):
+    def pred_pts_seg(self, value: PointData) -> None:
         self.set_field(value, '_pred_pts_seg', dtype=PointData)
 
     @pred_pts_seg.deleter
-    def pred_pts_seg(self):
+    def pred_pts_seg(self) -> None:
         del self._pred_pts_seg
 
+    @property
```

Comment:
For now, we have not decide whether to make eval_ann_info an attribute of the data sample. I suggest we omit lines 234-247 and their corresponding unit tests in this PR and do it in the future.

Reply:
removed. I have another question, whether should we implement `cat` method for `PointData`.